### PR TITLE
Class ModelDeployment

### PR DESCRIPTION
**Pull request by HermanPlay**

### CODE REVIEW

File: app/api/utils/model_deployment.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        self.cpu_limit: str = cpu_limit
+        self.cpu_request: str = cpu_request
+        self.memory_limit: str = memory_limit
+        self.memory_request: str = memory_request
+
+    def deploy(self):
+        """
+        Deploys a docker image to a kubernetes cluster.
+        """
+
```
Comment (matined) [inline app/api/utils/model_deployment.py:46]: don't put an empty line at the beginning of a method

File: app/api/utils/model_deployment.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        """
+        self.name: str = name
+        self.image_tag: str = image_tag
+        self.replicas: int = replicas
+        self.cpu_limit: str = cpu_limit
+        self.cpu_request: str = cpu_request
+        self.memory_limit: str = memory_limit
+        self.memory_request: str = memory_request
+
+    def deploy(self):
```
Comment (matined) [inline app/api/utils/model_deployment.py:42]: add return types everywhere

**Review by matined — CHANGES_REQUESTED** [review]
great job, two additional things:
1. Could you also add a static method that allows to delete an existing model deployment?
2. k8s folder can be removed from here, I'll pass it to devops team, so they can recreate the setup in deployment pipeline

**Comment by HermanPlay** [comment]
@matined Done

**Review by matined — APPROVED** [review]

**Review by matined — APPROVED** [review]

**Pull request merged**